res/css/style.css

```diff
@@ -194,6 +194,9 @@ body,
   background-color: var(--grey-30);
   color: black;
 }
+.treeViewRow.isSelected a {
+  color: white;
+}
```

Comment:
As you see below, the "highlight" background color is set only when there's a focus. Therefore the color needs to be set only in this case. Then the selector would be `.treeViewBody:focus .treeViewRow.isSelected.

Also I think the color could be `inherit` (so that it's the same as the rest of the row). Which gives:
```
.treeViewBody:focus .treeViewRow.isSelected a {
  color: inherit;
}
```
Also please put it closer to the other similar selector below.

Additional notes:
I thought of using `inherit` for every links, but actually I believe it's better to keep it in blue for other cases.
Also I don't like this selector much (2 "descendant" selector, and the `a` that can apply to all links in the app), but it shouldn't be too bad thanks to the bloom filter and the fact we don't have a lot of links in the app anyway.